checkstyle-tester/diff.groovy

```diff
@@ -289,6 +306,27 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     println 'Creating report summary page finished...'
 }
 
+def generateConfigFileHtml(diffDir, configFile, textTransform, summaryIndexHtml) {
```

Comment:
Since this isn't returning anything it should be renamed to something like `generateAndPrintConfigHtmlFile` to denote it is printing the file to the summary now and not just generating the file.

Reply:
Yes, I also thought about this. Fixed.